services/management/ia/rules_service.go

```diff
@@ -92,8 +92,22 @@ type rule struct {
 	Annotations map[string]string   `yaml:"annotations,omitempty"`
 }
 
+// RemoveVMAlertRulesFiles removes all generated rules files (*.yml) on the ia path.
+func (s *RulesService) RemoveVMAlertRulesFiles() {
+	matches, err := filepath.Glob(s.rulesPath + "/*.yml")
+	if err != nil {
+		s.l.Errorf("Failed to clean old alert rule files: %+v", err)
```

Comment:
it's better to return some error here, not just log and go ahead

Reply:
Done